pai-management/paictl.py

```diff
@@ -110,8 +110,8 @@ def generate_secret_base64code(docker_info):
 
 def generate_docker_credential(docker_info):
```

Comment:
Not the latest code? Please see: https://github.com/Microsoft/pai/blob/b6161ef164abc4d60916c7e54615786b4b564816/pai-management/paictl.py#L85

Reply:
Please merge the latest code from `management-dev` branch and have a try again.